pkg/controller/acmechallenges/controller.go

```diff
@@ -140,26 +144,18 @@ func (c *controller) Register(ctx *controllerpkg.Context) (workqueue.RateLimitin
 	// read options from context
 	c.dns01Nameservers = ctx.ACMEOptions.DNS01Nameservers
 	c.DNS01CheckRetryPeriod = ctx.ACMEOptions.DNS01CheckRetryPeriod
-
+	c.maxChallengesPerSchedule = ctx.ACMEOptions.MaxChallengesPerSchedule
 	return c.queue, mustSync, nil
 }
 
-// MaxChallengesPerSchedule is the maximum number of challenges that can be
-// scheduled with a single call to the scheduler.
-// This provides a very crude rate limit on how many challenges we will schedule
```

Comment:
Can we document this line somewhere, even I was a bit confused what a `max-challenges-per-schedule` would do and I am the maintainer of this project :sweat_smile:

Reply:
Sure thing, It's now on the context field, controller field and part of the flag help text